### PR TITLE
Ticket5009 muon jaws

### PR DESCRIPTION
**Pull request by davidkeymer**
Support module for the muon jaws IOC

### CODE REVIEW

File: MuonJawsApp/protocol/lv_MuonJaws.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Initially generated by $Id$ -->
+<lvinput xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://epics.isis.rl.ac.uk/lvDCOMinput/1.0" xsi:schemaLocation="http://epics.isis.rl.ac.uk/lvDCOMinput/1.0 lvDCOMinput.xsd">
+  <!--Full path to external interface vi used for triggering LabVIEW events when extint="true" below -->
+  <extint path="$(LVDCOM=$(TOP))/lvDCOMApp/src/extint/Main/Library/External Interface - Set Value.vi">
+    <!--Look for extint in LVDCOM directory or TOP if not defined-->
+  </extint>
+  <!--Definition of a configSection, the second argument of the IOC lvDCOMConfigure() command -->
+  <section name="frontpanel">
+    <!--TODO: edit path attribute in <vi /> to contain a full absolute path, you can also use any EPICS macro known to the IOC e.g. $(TOP) or $(LVDCOM)-->
```
Comment (JamesKingWork) [inline MuonJawsApp/protocol/lv_MuonJaws.xml]: Why do we have todos in here?

File: iocBoot/iocMuonJaws/config.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+<config_part>
```
Comment (JamesKingWork) [inline iocBoot/iocMuonJaws/config.xml]: ioc_desc?

File: iocBoot/iocMuonJaws/config.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+<macros>
+<macro name="LVDCOM_OPTIONS" pattern="^\d?\d?$" description="Options that define how the VI is started. Add selected options together to make the final value. 1 - warn if idle, 2 - start if idle, 4 - stop VIs if started on exit, 8 - stop VI on exit. (Default 1)" />
```
Comment (JamesKingWork) [inline iocBoot/iocMuonJaws/config.xml]: Should this not contain the regular things such as PORT, BAUD, BITS, PARITY etc.

**Review by JamesKingWork — COMMENTED** [review]
A few minor comments more as questions than requested changes. I also wanted to ask if it was possible to test using the IocTestFramework an LVDCOM IOC, I am presuming not, but, if possible, we should add some.

File: iocBoot/iocMuonJaws/config.xml (hunk 2 above)
Comment (davidkeymer) [reply iocBoot/iocMuonJaws/config.xml]: The support module should not contain an iocBoot area - I forgot to remove it after auto-generating the directory structure.

File: iocBoot/iocMuonJaws/config.xml (hunk 3 above)
Comment (davidkeymer) [reply iocBoot/iocMuonJaws/config.xml]: See comment above.

File: MuonJawsApp/protocol/lv_MuonJaws.xml (hunk 1 above)
Comment (davidkeymer) [reply MuonJawsApp/protocol/lv_MuonJaws.xml]: Fair point - I forgot to remove it!

**Review by JamesKingWork — APPROVED** [review]

**Pull request merged**